Eatery/Views/Eateries/HistogramViewController.swift

```diff
@@ -0,0 +1,285 @@
+//
+//  HistogramViewController.swift
+//  Eatery
+//
+//  Created by Ethan Fine on 4/26/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+// TODO: Create a protocol that encapsulates all Swipe Data and make HistogramViewController independent of Eatery variables
+class HistogramViewController: UIViewController {
+    
+    // MARK: Data
+    var swipeData: EaterySwipeData!
```

Comment:
`private let swipeData: EaterySwipeData`. 

We're passed it in the initializer. 

Edit: And in general, we should find ways to make as many of the non-view properties below constants and private.